src/Symfony/Component/Notifier/Exception/UnsupportedSchemeException.php

```diff
@@ -152,6 +152,10 @@ class UnsupportedSchemeException extends LogicException
             'class' => Bridge\Zulip\ZulipTransportFactory::class,
             'package' => 'symfony/zulip-notifier',
         ],
+        'telnyx' => [
+            'class' => Bridge\Telnyx\TelnyxTransportFactory::class,
+            'package' => 'symfony/telnyx-notifier',
+        ],
```

Comment:
Shouldn't it be ordered alphabetically? cc @OskarStark

Reply:
yes please

Reply:
updated